contracts/src/BackedCommunityStorageV1.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.12;
+
+import {IBackedCommunityTokenDescriptorV1} from "./interfaces/IBackedCommunityTokenDescriptorV1.sol";
+import {IBackedCommunityTokenV1} from "./interfaces/IBackedCommunityTokenV1.sol";
+
+contract BackedCommunityStorageV1 {
+    // @notice keep track of the total number of community NFTs minted
+    uint256 public nonce;
```

Comment:
Agree that `totalSupply` is a better name here.

The NFTs don't need an incremental ID when minting unless there's a need to iterate over all minted NFTs. You could just use the address as the ID?